openpype/hosts/maya/plugins/publish/validate_rendersettings.py

```diff
@@ -239,6 +239,15 @@ def get_invalid(cls, instance):
                 "{}_render_attributes".format(renderer)) or []
         )
 
+        settings_lights_flag = instance.context.data["project_settings"].get(
+            "maya", {}).get(
+            "create", {}).get(
+            "CreateRender", {}).get(
+            "enable_all_lights", {})
+
+        instance_lights_flag = instance.data.get("RenderSetupIncludeLights")
+        if settings_lights_flag != instance_lights_flag:
+            cls.log.warning("Warning: render settings differ from instance")
```

Comment:
I thinks this message should also explain why.

Reply:
This should work now, was missing variable collection from the collector